app1/admin.py

```diff
@@ -1,3 +1,19 @@
-from django.contrib import admin
+#from django.contrib import admin
 
 # Register your models here.
+from django.contrib import admin
+from .models import YourModel  # Import your model(s) here
+
+@admin.register(YourModel)
```

Comment:
**Suggestion**: Add docstrings to describe the purpose of the class 'YourModelAdmin' for better code documentation. <br> **Code Suggestion**: 
 ```
 +"""Register YourModelAdmin class for admin interface"""
 +@admin.register(YourModel)
 ```